src/lib/i18n.ts

```diff
@@ -0,0 +1,82 @@
+import i18next from 'i18next';
+import { createI18nStore } from 'svelte-i18next';
+
+i18next.init({
+	lng: 'en',
+	resources: {
+		en: {
+			translation: {
+				session_one: 'Session',
+				session_other: 'Sessions',
+				knowledge_one: 'Knowledge',
+				knowledge_other: 'Knowledge',
+				settings: 'Settings',
+				motd: 'Message of the day',
+				sidebar: {
+					sessions: 'Sessions',
+					knowledge: 'Knowledge',
+					settings: 'Settings',
+					motd: 'Motd',
```

Comment:
Let's not repeat keys with the same value, grab them from the `sessions` and `knowledge` keys.